### PR TITLE
Fix compilation problems & disable fastflash

### PR DESCRIPTION
**Pull request by jhoenicke**

### CODE REVIEW

**Comment by saleemrashid** [comment]
Fastflash is still useful for testing bootloader changes.

File: emulator/setup.c [hunk 1]
```diff
@@ -48,6 +48,10 @@ void setup(void) {
 	setup_flash();
 }
 
+void __attribute__((noreturn)) shutdown(void) {
+	for(;;);
```
Comment (saleemrashid) [inline emulator/setup.c]: `for(;;) pause();` seems more appropriate here.

**Comment by jhoenicke** [comment]
@saleemrashid But fastflash can't work with newer bootloaders, right?  It assumes running in privileged mode and that is not available for unsigned firmware.  Of course, you can keep a Trezor with a 1.3.4 bootloader for testing other bootloaders.

File: emulator/setup.c (hunk 1 above)
Comment (jhoenicke) [reply emulator/setup.c]: Fixed that.  But in the long run, we may think of reworking also emulatorPoll, to not hog the CPU in normal mode.

**Comment by saleemrashid** [comment]
My mistake, I didn't think about that :)

**Comment by prusnak** [comment]
Thanks!

**Pull request merged**